daps-xslt/contrib/convert-table2variablelist.xsl

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Purpose:
+     Converts a informaltable or table into a variablelist.
+
+   Parameters:
+     * None
+
+   Input:
+     DocBook 5 document
+
+   Output:
+     DocBook 5 document with variablelist.
+
+   Design:
+     Any elements inside entry/para are copied to a listitem. It is
+     assumend, that both content models are the same.
+
+   HINTS:
+     * If you want to keep entities, you need to protect them before the
+       transformation.
```

Comment:
"...transformation, for example: &foo; > [[foo]]"

Reply:
Added into commit 823a3d3b. Also added a reference to `libexec/entities-exchange.sh`